delegation.go

```diff
@@ -0,0 +1,62 @@
+	DelegatorABI = `[{"constant":false,"inputs":[{"name":"_target","type":"address"}],"name":"setTarget","outputs":[],"payable":false,"stateMutability":"nonpayable","type":"function"},{"constant":true,"inputs":[],"name":"getDelegateStorage","outputs":[{"name":"","type":"address"}],"payable":false,"stateMutability":"view","type":"function"},{"constant":true,"inputs":[],"name":"getTarget","outputs":[{"name":"","type":"address"}],"payable":false,"stateMutability":"view","type":"function"},{"payable":false,"stateMutability":"nonpayable","type":"fallback"}]`
+
+	StorageBin = `0x608060405234801561001057600080fd5b5061011f806100206000396000f300608060405260043610603f576000357c0100000000000000000000000000000000000000000000000000000000900463ffffffff1680639bb66b28146044575b600080fd5b348015604f57600080fd5b5060c8600480360381019080803573ffffffffffffffffffffffffffffffffffffffff169060200190929190803590602001908201803590602001908080601f016020809104026020016040519081016040528093929190818152602001838380828437820191505050505050919291929050505060ca565b005b600080825160208401856127105a03f43d604051816000823e826000811460ef578282f35b8282fd00a165627a7a7230582066e5f062551aab8c16aaac9d9eb84dd5138b2ea37a44d1cd624ac6a075e6f1cc0029`
+	StorageABI = `[{"constant":false,"inputs":[{"name":"target","type":"address"},{"name":"msgdata","type":"bytes"}],"name":"invoke","outputs":[],"payable":false,"stateMutability":"nonpayable","type":"function"}]`
+)
+
+// DelegatorCode returns delegation code with a given storage address and target contract.
+func DelegatorCode(storage, target common.Address) string {
+	code := DelegatorBin
+
+	// Replace placeholder addresses for storage & target contract.
+	code = strings.Replace(code, `aabbaabbaabbaabbaabbaabbaabbaabbaabbaabb`, strings.TrimPrefix(storage.String(), "0x"), -1)
+	code = strings.Replace(code, `eeffeeffeeffeeffeeffeeffeeffeeffeeffeeff`, strings.TrimPrefix(target.String(), "0x"), -1)
```

Comment:
Since solidity is available to us, we could do these replacements in the source and then recompile. A little safer and we'd pick up new compiler changes automatically, but not a big deal.